common/types.go

```diff
@@ -478,3 +478,14 @@ func (l Location) SubLocation() int {
 		return -1
 	}
 }
+
+func (l Location) IntArrayLocation() []int {
```

Comment:
byte in golang are uint8. Is there really a need for this helper function?

Reply:
There was other conversation on this, but yes because it is needed to avoid base-64 encoded strings.

Reply:
Fixed now for a re-review.